bitnami/mariadb-galera/values.yaml

```diff
@@ -178,6 +178,22 @@ ldap:
   scope:
   tls_reqcert:
 
+## SSL configuration
+##
+ssl:
+  ## Replicaction traffic
+  rt:
+    ## Enable SSL for replication traffic
+    enabled: false
+    ## Name of the secret that contains the certificates
+    certificatesSecret:
+    ## Certificate filename
+    certFilename:
+    ## Certificate Key filename
+    certKeyFilename:
+    ## CA Certificate filename
+    certCAFilename:
```

Comment:
General conventions:

Use:

```
## Some comment
##
key: value
```

Instead of:

```
## Some comment
key: value
```

When you're not setting any default value, comment the key, instead of setting it to an empty value:

```diff
- key:
+ # key
```